### ISSUE TITLE
Training Strategy

### ISSUE THREAD

**Issue by StephCookkfjdkjh** [YihangChen-ee/FCGS#14]
Hi, Chen. Thanks for your insightful work. I have some question about the training. 

1. How many iterations did you take to train the network, especially on the first stage of the m=1 autoencoder part. How many epochs did you implement on the 6000+ Gaussian samples, to achieve the PSNR 28.82dB as reported? 
2. Did you use any other training strategy other than including mask ratio into the loss? 

I highly look forward to your reply.

**Comment by YihangChen-ee** [comment]
Hi,

1.	Given that the dataset contains over 6000 Gaussian scenes, training the first stage for around 50 epochs generally yields good results. In practice, we tuned the model while training, so it’s hard to give an exact number of epochs. But training for 50 epochs or more in the first stage should be sufficient to reach a decent performance.

2.	Regarding the training strategy, you may refer to [this issue](https://github.com/YihangChen-ee/FCGS/issues/11). BTW, in Stage 2, you may consider retaining the mask ratio loss (with $\lambda_m$) during the initial training steps, since the entropy MLPs in $m=1$ tends to converge slower than entropy MLPs in $m=0$, and the same collapse problem will raise if such mask ratio loss is not utilized. After several epochs in Stage 2, you can gradually remove the mask ratio loss, so that the training is eventually a pure RD loss.

**Comment by StephCookkfjdkjh** [comment]
Thanks for your kind rely. That helps with my understanding.
